core/src/main/java/com/linecorp/armeria/server/AnnotatedServiceBindingBuilder.java

```diff
@@ -0,0 +1,213 @@
+        return (AnnotatedServiceBindingBuilder) super.requestContentPreviewerFactory(factory);
+    }
+
+    @Override
+    public AnnotatedServiceBindingBuilder responseContentPreviewerFactory(ContentPreviewerFactory factory) {
+        return (AnnotatedServiceBindingBuilder) super.responseContentPreviewerFactory(factory);
+    }
+
+    @Override
+    public AnnotatedServiceBindingBuilder contentPreview(int length) {
+        return (AnnotatedServiceBindingBuilder) super.contentPreview(length);
+    }
+
+    @Override
+    public AnnotatedServiceBindingBuilder contentPreview(int length, Charset defaultCharset) {
+        return (AnnotatedServiceBindingBuilder) super.contentPreview(length, defaultCharset);
+    }
+
+    @Override
+    public AnnotatedServiceBindingBuilder contentPreviewerFactory(ContentPreviewerFactory factory) {
+        return (AnnotatedServiceBindingBuilder) super.contentPreviewerFactory(factory);
+    }
+
+    @Override
+    public AnnotatedServiceBindingBuilder accessLogFormat(String accessLogFormat) {
+        return (AnnotatedServiceBindingBuilder) super.accessLogFormat(accessLogFormat);
+    }
+
+    @Override
+    public AnnotatedServiceBindingBuilder accessLogWriter(AccessLogWriter accessLogWriter,
+                                                          boolean shutdownOnStop) {
+        return (AnnotatedServiceBindingBuilder) super.accessLogWriter(accessLogWriter, shutdownOnStop);
+    }
+
+    /**
+     * Registers the {@link ServiceConfigBuilder} that is created from this Builder and returns
+     * {@link ServerBuilder}.
+     * @param service annotated service object to handle incoming requests matching path prefix, which
+     *                  can be configured through {@link AnnotatedServiceBindingBuilder#pathPrefix(String)}.
+     *                  If path prefix is not set then this service is registered to handle requests matching
+     *                  {@code /}
+     * @return {@link ServerBuilder} to continue building {@link Server}
+     */
+    public ServerBuilder buildAnnotated(Object service) {
+        final ImmutableList<ExceptionHandlerFunction> exceptionHandlerFunctions =
+                exceptionHandlerFunctionBuilder.build();
+        final ImmutableList<RequestConverterFunction> requestConverterFunctions =
+                requestConverterFunctionBuilder.build();
+        final ImmutableList<ResponseConverterFunction> responseConverterFunctions =
+                responseConverterFunctionBuilder.build();
+
+        final Iterable<?> exceptionHandlersAndConverters = Iterables.concat(exceptionHandlerFunctions,
+                                                                            requestConverterFunctions,
+                                                                            responseConverterFunctions);
+        final List<AnnotatedHttpServiceElement> elements =
+                AnnotatedHttpServiceFactory.find(pathPrefix, service, exceptionHandlersAndConverters);
```

Comment:
How about adding a new find method because the find method split the merged list(`exceptionHandlersAndConverters`) again. It seems inefficient.

```java
public static List<AnnotatedHttpServiceElement> 
        find(String pathPrefix, Object object,
	         List<RequestConverterFunction> exceptionHandlersAndConverters,
	         List<RequestConverterFunction> requestConverterFunctions,
	         List<ResponseConverterFunction> responseConverterFunctions) {
  ...
}
```